.github/.coderabbit.yaml

```
@@ -0,0 +1,129 @@
+      disabled_categories: [ TYPOS, TYPOGRAPHY, CASING ]
+
+chat:
+  auto_reply: true
+
+knowledge_base:
+  opt_out: false
+  learnings: { scope: auto }
+  issues:   { scope: auto }
+  jira:     { project_keys: [] }
+  linear:   { team_keys: [] }
+  pull_requests: { scope: auto }
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)